source/Mlos.Agent.Server/Mlos.Agent.Server.csproj

```diff
@@ -12,6 +12,10 @@
   </PropertyGroup>
   <Import Project="$(BaseDir)\build\Mlos.NetCore.props" />
   <Import Project="$(BaseDir)\build\Mlos.NetCore.SingleFile.props" />
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" GeneratePathProperty="true" />
+    <BinplaceFile Include="$(PkgCommandLineParser)/lib/netstandard2.0/CommandLine.dll" />
```

Comment:
@grlap and I have discussed changing these later to provide a directory of symlinks to the output dir for convenience instead of binplacing everything in `target/`.

For now, I'm just sticking with the existing binplace method.  Will circle around and clean that up further later.

Reply:
To unblock myself, I removed binplacing in my private branch.